demisto_sdk/commands/create_artifacts/tests/contet_creator_test.py

```diff
@@ -0,0 +1,31 @@
+import filecmp
+from demisto_sdk.commands.create_artifacts.content_creator import *
+
+
+def test_copy_dir_files():
+    current_dir = os.path.dirname(__file__)
+    content_repo_example = os.path.join(current_dir, '/test_files/content_repo_example')
```

Comment:
you already using join...
```suggestion
    content_repo_example = os.path.join(current_dir, 'test_files', 'content_repo_example')
```